projects/fine_tuning/toolbox/fine_tuning_run_fine_tuning_job/files/entrypoint/run_fms.sh

```diff
@@ -19,16 +19,3 @@ if [[ $WORLD_SIZE == 1 ]]; then
     time python /app/accelerate_launch.py
     exit 0
 fi
-echo "Running on $WORLD_SIZE machines with $NUM_GPUS GPUs each."
```

Comment:
Hey Alberto 👋🏻 

I think you should update the `if` above, to have something like:
```
if [[ $WORLD_SIZE != 1 ]]; then
  echo "Running with a multi-node configuration. This is not supported at the moment, aborting."
  exit 1
fi
```
and move everything from the current `if` branch one level below